content/4_Gold/Knapsack_DP.mdx

```diff
@@ -134,6 +134,15 @@ export const problems = {
 			['Knapsack', 'Prime Factorization'],
 			'Prime factorize $K$.'
 		),
+		new Problem(
+			'CSES',
+			'Counting Numbers',
```

Comment:
I don't think this counts as knapsack DP?

Reply:
should i put this in the nt module since dp-nt is removed?

Reply:
I wouldn't count this as NT

Reply:
where should i put it then? :/

Reply:
nowhere I guess, haven't seen digit DP on USACO :////

Reply:
perhaps if we find more dp digit problems